lib/hutch/broker.rb

```diff
@@ -219,12 +220,12 @@ def publish(routing_key, message, properties = {}, options = {})
         "publishing #{spec} to #{routing_key}"
       }
 
-      response = @exchange.publish(payload, {persistent: true}.
+      response = publish_exchange.publish(payload, {persistent: true}.
```

Comment:
This name makes little sense. Exchanges are not used for anything besides publishing.